### PR TITLE
Ds client upgrade

### PR DESCRIPTION
**Pull request by ethervoid**
Deploy it after the deploy the client version in all user DBs

// I've changed Gruntfile to check for 2.14 and 2.15 npm version because nvm install the 2.15 for the 0.10.x version of node. What do you think @xavijam ?

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
  var jasmineCfg = require('./lib/build/tasks/jasmine.js');
  var duplicatedDependencies = require('./lib/build/tasks/shrinkwrap-duplicated-dependencies.js');
 
- var REQUIRED_NPM_VERSION = /2.14.[0-9]+/;
+ var REQUIRED_NPM_VERSION = /2.1[4-5].[0-9]+/;
```
Comment (xavijam) [inline Gruntfile.js]: Uhm, I don't remember exactly why we set this specific version, but there was a reason behind. Do you remember it @rochoa ?

**Comment by ethervoid** [comment]
Removed the Gruntfile part to avoid problems

File: Gruntfile.js (hunk 1 above)
Comment (rochoa) [inline Gruntfile.js]: We want to keep npm-shrinkwrap as similar as possible. Using different versions of npm 2.x series was creating different npm-shrinkwrap files, making the review process a bit painful.

@ethervoid, you can force a version by doing `npm install -g npm@2.14.x`, that way you don't have to keep switching it with nvm.

Comment (ethervoid) [inline Gruntfile.js]: Understood, makes sense!

Thank you for the tip to for version 2.14 :) :)

**Pull request merged**